### PR TITLE
:arrow_up::one::two: group_menu_no_access *Port*

### PR DESCRIPTION
**Pull request by Ommo73**

### CODE REVIEW

File: group_menu_no_access/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+======================
+ Group Menu No Access
+======================
+
+Installation
+============
+
+* `Install <https://odoo-development.readthedocs.io/en/latest/odoo/usage/install-module.html>`__ this module in a usual way
```
Comment (yelizariev) [inline group_menu_no_access/doc/index.rst]: Add usage instruction

File: group_menu_no_access/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Configuration
+=============
+
+Does not require pre-configuration
```
Comment (yelizariev) [inline group_menu_no_access/doc/index.rst]: It does.  Read explanation in template for understanding difference between Configuration and Usage
https://odoo-development.readthedocs.io/en/latest/dev/docs/usage-instructions.html

File: group_menu_no_access/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+Usage
+=====
+
+* Open menu ``Settings >> Users & Companies >> Groups``
+* Select the group for which you want to change permissions
+* Click on ``[Edit]`` button
+* Go to the ``[Menus]`` tab
+* Set permissions for this user group
```
Comment (yelizariev) [inline group_menu_no_access/doc/index.rst]: Which permisions? The form has bunch of settings

File: group_menu_no_access/doc/index.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+* Click on ``[Save]`` button
```
Comment (yelizariev) [inline group_menu_no_access/doc/index.rst]: Ok, and what we get? Why do we need to do those settings? How to check that it works?

File: group_menu_no_access/doc/index.rst (hunk 2 above)
Comment (Ommo73) [reply group_menu_no_access/doc/index.rst]: Configuration added

File: group_menu_no_access/doc/index.rst (hunk 4 above)
Comment (Ommo73) [reply group_menu_no_access/doc/index.rst]: Added description of settings and results

File: group_menu_no_access/doc/index.rst [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+Usage
+=====
+
+* Open menu `Settings >> Users & Companies >> Groups`
+* Select the group for which you want to change permissions to access for menu and sub-menu items
+* Click on `[Edit]` button
+* Go to the `[Menus]` tab
+* Set permissions for this user group by adding to the table "Deny Access" sections to which you want to close access to the user and in the table "Allow Access" sections to which you want to open
+* Click on `[Save]` button
+* RESULT: sections closed to assigned users are invisible now and they can't access to these sections with the direct link
```
Comment (yelizariev) [inline group_menu_no_access/doc/index.rst]: RESULT is the only text for USAGE section here

**Comment by yelizariev** [comment]
I approve to merge it now

**Pull request merged**